### PR TITLE
Add photutils

### PR DESCRIPTION
**Pull request by mwcraig**
This PR adds the package photutils to conda-forge.

@larrybradley let me know if you would like to be added as a maintainer on this. Going forward the astropy conda channel will be populated with builds from conda-forge instead of maintaining our own build infrastructure.

### CODE REVIEW

File: recipes/photutils/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    - photutils.geometry
+    - photutils.morphology
+    - photutils.psf
+    - photutils.psf.matching
+    - photutils.segmentation
+    - photutils.utils
+
+about:
+  home: http://photutils.readthedocs.io/
+  license: BSD 3-caluse
```
Comment (isuruf) [inline recipes/photutils/meta.yaml]: `clause`

File: recipes/photutils/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+build:
+  number: 0
+  script: python setup.py install --offline --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - astropy
```
Comment (isuruf) [inline recipes/photutils/meta.yaml]: Is astropy needed at build time?

**Comment by mwcraig** [comment]
Thanks for the review, @isuruf. Turns out `astropy` is not a build dependency, but it should have required `numpy x.x` and `cython` at build.

**Comment by larrybradley** [comment]
Thanks, @mwcraig!  Yes, please add me as a maintainer.

File: recipes/photutils/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+build:
+  number: 0
+  script: python setup.py install --offline --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy x.x
```
Comment (ocefpaf) [inline recipes/photutils/meta.yaml]: Do you want to try https://github.com/conda-forge/conda-forge.github.io/pull/415 here @mwcraig? Or should we merge this?

Comment (larrybradley) [inline recipes/photutils/meta.yaml]: @mwcraig  `Photutils` requires `numpy >= 1.9`.

Comment (mwcraig) [inline recipes/photutils/meta.yaml]: @ocefpaf said:

> Do you want to try conda-forge/conda-forge.github.io#415 here @mwcraig?

Sure; we've been shipping unpinned versions of this for about 1.5 years with no complaints.

File: recipes/photutils/meta.yaml [hunk 4]
```diff
@@ -20,12 +20,13 @@ requirements:
   build:
     - python
     - setuptools
-    - numpy x.x
+    - numpy 1.9.*  # [not py36]
```
Comment (ocefpaf) [inline recipes/photutils/meta.yaml]: If this works with np18 you can use it for all python's. (I added np18 last week,)

Comment (mwcraig) [inline recipes/photutils/meta.yaml]: Per @larrybradley it requires numpy >=1.9

Comment (ocefpaf) [inline recipes/photutils/meta.yaml]: Sorry. I hit send before finishing my comment. I also added np19. You can use that for all Python's too.

Comment (mwcraig) [inline recipes/photutils/meta.yaml]: Is 1.9 available on python 3.6? Right, I see it there now. Pushing the change...

Comment (ocefpaf) [inline recipes/photutils/meta.yaml]: > Is 1.9 available on python 3.6? Right

And I am working on np17 too, but that may take a while (and very few projects will use it).

**Comment by mwcraig** [comment]
@ocefpaf -- looks like there is no numpy 1.9/python 3.6 on Windows. Is that coming or should I change the version in the recipe?

**Comment by ocefpaf** [comment]
> @ocefpaf -- looks like there is no numpy 1.9/python 3.6 on Windows.

Sorry. I misjudged that one. (I always forget about Windows.)

> Is that coming or should I change the version in the recipe?

Yes and yes. I am working on it but it will take some time, so I do recommend to adapt the recipe accordingly until I finish it. Sorry...

**Comment by mwcraig** [comment]
All review comments have been addressed, merging...someone please let me know if I shouldn't be self-merging 😮

**Pull request merged**